canonical-web-and-design/ubuntu.com
# Update /kubernetes/install page

## Done

- Updated /kubernetes/install page

## QA

- Download the branch
- Go to http://0.0.0.0:8001/kubernetes/install
- See that the page matches the [copy doc](https://docs.google.com/document/d/1oiD6UUomf2daZitS3UOosujYBXPbiOMzPtXVZBq-ph0/edit#)

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1256
